examples/planning/closed_loop_test.ipynb

```diff
@@ -298,27 +317,25 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Visualise the closed-loop\n",
+    "## Visualise the closed-loop\n",
     "\n",
-    "We can visualise one of the example we have stored in the previous cell. \n",
+    "We can visualise the scenes we have obtained previously. \n",
     "\n",
     "**The policy is now in full control of the SDV as this moves through the annotated scene.**"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": false
+   },
    "outputs": [],
    "source": [
-    "from IPython.display import display, clear_output\n",
-    "import PIL\n",
-    "from time import sleep\n",
-    "\n",
-    "for frame in images[0]:\n",
-    "    clear_output(wait=True)\n",
-    "    display(PIL.Image.fromarray(frame))\n",
-    "    sleep(0.05)"
+    "output_notebook()\n",
+    "for sim_out in sim_outs: # for each scene\n",
+    "    vis_in = simulation_out_to_visualizer_scene(sim_out, mapAPI)\n",
```

Comment:
with the changes introduced in #277 you can now get a `mapAPI` object regardless of the rasteriser (`mapAPI = rasterizer.sem_rast.mapAPI` only works with a `SemanticRasterizer`) using the `mapAPI.from_cfg` static method. Would be nice to replace it here